.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: CI Checks
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Check audit, lint, and prettier
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+      - run: npm install
```

Comment:
It might be worthwhile to have this command be `npm ci` since there are some small benefits to using that command and this is a new file.

Reply:
It's a good idea, but apparently `npm ci` is more sensitive to the `package-lock.json` format -- because it causes the Node 14 actions to fail. Since we still want to target Node 14 (for now), I think we need to stick with `npm install`.

Reply:
Agreed!